constructorio-client/src/main/java/io/constructor/client/models/QuizQuestionResponse.java

```diff
@@ -0,0 +1,33 @@
+package io.constructor.client.models;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Constructor.io Quiz Question Response ... uses Gson/Reflection to load data in
+ */
+public class QuizQuestionResponse {
+
+    @SerializedName("next_question")
+    private QuizQuestion quizQuestion;
+
+    @SerializedName("is_last_question")
+    private Boolean isLastQuestion;
+
+    @SerializedName("version_id")
+    private String versionId;
+
+    /**
+     * @return the next_question
+     */
+    public QuizQuestion getQuizQuestion() { return quizQuestion; }
```

Comment:
Looks like we'll be renaming the function to `getQuizNextQuestion`. Let's also change `quizQuestion` variable back to `nextQuestion` to be in line with the response field